virtio-queue/src/descriptor_utils.rs

```diff
@@ -0,0 +1,880 @@
+// Copyright 2019 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Maybe we should follow the other files.

What about this changes?

```suggestion
// Portions Copyright 2019 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE-BSD-3-Clause file.
```

Reply:
Feel free to add also `// Copyright (C) 2024 Red Hat, Inc. All rights reserved.`

Reply:
Done, thanks.